app/src/main/res/values/colors.xml

```diff
@@ -27,13 +27,12 @@
 
     <color name="pocket_onboarding_background">#e60c0c0d</color>
 
-    <color name="button_highlight">#64ffffff</color>
-    <color name="button_default">@color/translucent_tv_ink</color>
-    <color name="button_disabled">@color/tv_ink</color>
-    <color name="button_focused">@color/tv_blue50</color>
+    <color name="button_highlight">#00ffffff</color>
+    <color name="button_default">@android:color/transparent</color>
```

Comment:
I'm a little on the fence about whether we should wrap/alias these if they're just `transparent`.

On the one hand, if we decide to change the button default color from transparent, we can just change it here and it does the right thing. But on the other hand, aliasing a color to transparent doesn't really make a ton of sense because we don't "need" the color to be defined in `colors.xml` we already know what it is.

I took a quick look at Focus, and I don't see any colors that wrap `transparent`, so I'm going to say that we should follow the pattern and not include transparent aliases in `colors.xml` but just use them directly.